pkg/cmd/roachtest/version.go

```diff
@@ -182,7 +172,9 @@ func registerVersion(r *testRegistry) {
 				if err := stop(i); err != nil {
 					return err
 				}
-				c.Put(ctx, b, "./cockroach", c.Node(i))
+				if err := c.Stage(ctx, c.l, "release", "v"+binaryVersion, "", c.Node(i)); err != nil {
```

Comment:
Doesn't this download the same binary in a loop? Can `c.Stage` or even `roachprod stage` do the caching so we don't have to be smart about each invocation of `c.Stage`?